nessie/merged/student_terms.py

```diff
@@ -32,25 +32,28 @@
 
 
 def upload_student_term_maps(advisees_by_sid):
-    (enrollment_terms_map, canvas_site_map) = generate_student_term_maps(advisees_by_sid)
+    # (enrollment_terms_map, canvas_site_map) = generate_student_term_maps(advisees_by_sid)
```

Comment:
@paulkerschen Is this comment left here on purpose?

Reply:
No!